core/src/main/java/com/scalar/db/common/StateManagedDistributedTransactionManager.java

```diff
@@ -144,9 +144,12 @@ public void commit() throws CommitException, UnknownTransactionStatusException {
 
     @Override
     public void rollback() throws RollbackException {
-      if (status == Status.COMMITTED || status == Status.ROLLED_BACK) {
+      if (status == Status.ROLLED_BACK) {
```

Comment:
`status` seems to move to ROLLED_BACK even when rolling back the transaction fails. If users first receive an exception from `rollback()` and then try the second attempt, it will look like success while the transaction might not be rolled back. What do you think?

Reply:
I see. Sounds good 👍